README.md

```diff
@@ -1,28 +1,279 @@
+| Trypsin+Chymotrypsin     |                                         |
+| Trypsin/P + ASP-N (D)    |                                         |
+| Asp-N(DE)                |                                         |
+| Trypsin/P+ASP-N(DE)      |                                         |
+| Trypsin/P+ASP-N(E)       |                                         |
+| Elastase                 |                                         |
+| Elastase & Trypsin       |                                         |
+| Trypsin/P & Exopeptidase |                                         |
+| Tryp-N                   |                                         |
+| No digestion             | for example used for synthetic peptides |
+
+
+
+#### Miscleavages
+The number of miscleavages to consider in the search. Given that crosslinked peptides generate spectra that are similar to spectra of long, miscleaved linear peptides, we suggest setting this number to 3, or even 4 if the database and set of modifications included in the search is small. This allows for alternative explanations of crosslinked spectra with miscleaved peptides.
+
+#### Number of Threads
+Number of threads to be used for the search. The memory usage scales with the number of threads. If the program runs out of memory, consider re-launching xiSEARCH with increased memory via the -Xmx option (see above) and/or reduce the number of threads.
+
+#### Modifications
+Modifications are considered in 3 flavours:
+
+1) Fixed: occurring on every instance of a residue
+2) Variable: may or may not be present on a residue
+3) Variable - linear peptides: may or may not be present on a residue, but will only be considered to explain spectra of non-crosslinked peptides. This option is used to simplify the search problem in searches with large (hundreds of proteins) databases.
+
+In the modifications tab, it is important to select the appropriate crosslinker modifications for your sample. In particular, loop modifications and hydrolysed crosslinkers are very common for both NHS-ester and diazirine crosslinkers. If a modification is selected in variable or variable-linear, it should not be selected in the other tab. Remember that the search problem (and therefore the memory and time necessary for the search) scales exponentially with the number of proteins in the database and their modifications.
+
+#### Ions
+Which fragment ion types to consider in the search.
+
+#### Losses
+Which fragment losses to consider in the search.
+
+### Custom config
+Here, additional configurations may be set using the text syntax as in the advanced config or in a config file used by the command line version of xiSEARCH (see next section)
+
+### Do FDR setting 
+If the "Do FDR" box is ticked, xiFDR will automatically be run at the end of xiSEARCH. We tend to leave this option off, as we prefer to run xiFDR in a standalone process to have access to more advanced FDR filtering options.
+
```

Comment:
```suggestion

### Changing or adding new entries to the config-gui
The  "BasicConfigEntries.conf" contains all the selectable config values. In this file new entries for crosslinker, enzymes, modifications and losses can be freely defined. The file contains sections for crosslinker, modifications, losses, ions,enzymes and custom settings. Each section has a short description on how to add new entries.

```

Reply:
Not sure if this is the right place to add this - but I think it should appear somewhere